ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
I moved this to scripts/manylinux/build_manylinux_wheel.sh and updated to be compatible with our Bamboo "run in docker" task